### PR TITLE
add tests for new subtitle conversions leveraging test substitle files from pressurecooker

### PR DESCRIPTION
**Pull request by ivanistheone**
enable tests while WIP ricecooker until pressucooker released

### CODE REVIEW

File: tests/test_files.py [hunk 1]
```diff
@@ -219,9 +221,83 @@ def test_youtubesubtitle_to_dict():
...
+
+
+
+# TODO: uncomment when edge case undetected ar fixed
+# def test_convertible_substitles_ar_ttml():
+#     assert os.path.exists("tests/testcontent/testsubtitles_ar.ttml")
+#     subtitle_file =  SubtitleFile("tests/testcontent/testsubtitles_ar.ttml", language='en-US')
+#     filename = subtitle_file.process_file()
+#     assert filename, 'conferted filename must exit'
+#     assert filename.endswith('.vtt'), 'conferted filename must have .vtt extension'
```
Comment (ivanistheone) [inline tests/test_files.py:302]: @bjester You can uncomment this when fix for `ar` subs being detected as `en-US` lands.
